components/cards/basic-cards/basic-cards.html

```diff
@@ -0,0 +1,186 @@
+                </div>
+            </div>
+        </div>
+        <!-- basic w/ text over media component2 ends -->
+
+        <!-- basic w/ header component3 starts -->
+        <div class="et_b-card-container et_b-wheader-card-container">
+            <div class="et_b-card-subcontainer1 et_b-wheader-card-subcontainer1">
+                <img />
+            </div>
+            <div class="et_b-card-subcontainer2 et_b-wheader-card-subcontainer2">
+                <h4 class="et_b-subcontainer2-heading1 et_b-wheader-card-subcontainer2-heading1">New album out now</h4>
+                <h6 class="et_b-subcontainer2-heading2 et_b-wheader-card-subcontainer2-heading2">By Alan Walker</h6>
+            </div>
+            <div class="et_b-card-subcontainer3 et_b-wheader-card-subcontainer3">
+                <p class="et_b-subcontainer3-para et_b-wheader-card-subcontainer3-para">Some of his top releases comprises lily, Alone, Darkside, Faded, Sing me
+                    to
+                    sleep, On my way etc.</p>
+            </div>
+            <div class="et_b-card-subcontainer4 et_b-wheader-card-subcontainer4">
+                <div class="et_b-subcontainer4-action-buttons et_b-wheader-card-subcontainer4-action-buttons">
+                    <button>READ</button>
+                    <button>BOOKMARK</button>
+                </div>
+                <div class="et_b-subcontainer4-icon-buttons et_b-wheader-card-subcontainer4-icon-buttons">
+                    <button><i class="far fa-heart"><span>like</span></i></button>
+                    <button><i class="fas fa-share-alt"><span>share</span></i></button>
+                    <button><i class="fas fa-ellipsis-v"><span>more options</span></i></button>
+                </div>
+            </div>
+        </div>
+        <!-- basic w/ header component3 ends -->
+
+        <!-- basic w/ only action buttons component4 starts -->
+        <div class="et_b-card-container et_basic-woab-card-container">
+            <div class="et_b-card-subcontainer1 et_basic-woab-subcontainer1">
+                <img />
+            </div>
+            <div class="et_b-card-subcontainer2 et_basic-woab-subcontainer2">
+                <h4 class="et_b-subcontainer2-heading1 et_basic-woab-subcontainer2-heading1">New album out now</h4>
+                <h6 class="et_b-subcontainer2-heading2 et_basic-woab-subcontainer2-heading2">By Alan Walker</h6>
+            </div>
+            <div class="et_b-card-subcontainer3 et_basic-woab-subcontainer3">
+                <p class="et_b-subcontainer3-para et_basic-woab-subcontainer3-para">Some of his top releases comprises lily, Alone, Darkside, Faded, Sing me
+                    to
+                    sleep, On my way etc.</p>
+            </div>
+            <div class="et_b-card-subcontainer4 et_basic-woab-subcontainer4">
+                <div class="et_b-subcontainer4-action-buttons et_basic-woab-subcontainer4-action-buttons">
+                    <button>READ</button>
```

Comment:
here you can use class for button to give css

Reply:
okay will try to resolve it!!, thankyou so much for reviewing!!